README.md

```diff
@@ -35,6 +35,22 @@ EventMachine.run do
 end
 ```
 
+To pass additional parameters while subscribing, add them to the identifier hash. These will be available server side in the channel's params hash.
```

Comment:
I think `channel` may be required by rails to know what channel you're sending/receiving on. (assuming you use channels)